source/integrations/setup-nginx-proxy-with-minio.rst

```diff
@@ -0,0 +1,193 @@
+======================================
+Configure Nginx Proxy for MinIO Server
+======================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+The following documentation covers the minimum settings required to configure Nginx to proxy requests to MinIO.
+
+This documentation assumes the following:
+
+- An existing `Nginx <http://nginx.org/en/download.html>`__ deployment
+- An existing :ref:`MinIO <minio-installation>` deployment
+- A DNS hostname which uniquely identifies the MinIO deployment
+
+There are two models for proxying requests to the MinIO Server API and the MinIO Console:
+
+.. tab-set::
+
+   .. tab-item:: Dedicated DNS
+
+      Create or configure a dedicated DNS name for the MinIO service.
+
+      For the MinIO Server S3 API, proxy requests to the root of that domain.
+      For the MinIO Console Web GUI, proxy requests to the ``/minio`` subpath.
+
+      For example, given the hostname ``minio.example.net``: 
+      
+      - Proxy requests to the root ``https://minio.example.net`` to the MinIO Server listening on ``https://minio.local:9000``.
+
+      - Proxy requests to the subpath ``https://minio.example.net/minio`` to the MinIO Console listening on ``https://minio.local:9001``.
+
+      The following location blocks provide a template for further customization in your unique environment:
+
+      .. code-block:: shell
+         :class: copyable
+
+         upstream minio {
+            least_conn;
+            server minio-01.internal-domain.com;
+            server minio-02.internal-domain.com;
+            server minio-03.internal-domain.com;
+            server minio-04.internal-domain.com;
+         }
+
+         server {
+            listen       80;
+            listen  [::]:80;
+            server_name  minio.example.net;
+
+            # To allow special characters in headers
```

Comment:
Drop the `To` in this and the other comment lines in the code sample.

`Allow special characters in headers` is fine.